src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/diff/PrisonerMovementsEventService.kt

```diff
@@ -24,31 +34,13 @@ class PrisonerMovementsEventService(
   ) {
     when (val movementChange = calculateMovementChange(previousPrisonerSnapshot, prisoner)) {
       PossibleMovementChange.None -> {}
-      is TransferIn -> domainEventEmitter.emitPrisonerReceiveEvent(
```

Comment:
all these case statements reduced to 2 because we have two separate sealed classes for movements in and out